### PR TITLE
Feature/intake3.0

### PR DESCRIPTION
**Pull request by NWSchallock**
PTAL

### CODE REVIEW

File: src/main/include/commands/IntakeOn.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * An example command.
+ *
+ * <p>Note that this extends CommandHelper, rather extending CommandBase
+ * directly; this is crucially important, or else the decorator functions in
+ * Command will *not* work!
+ */
+class IntakeOn
+    : public frc2::CommandHelper<frc2::CommandBase, IntakeOn> {
+ public:
+  IntakeOn(double speed, Intake& intake);
```
Comment (duane701) [inline src/main/include/commands/IntakeOn.h]: Subsystems goes first.

```suggestion
  IntakeOn(Intake& intake, double speed);
```

File: src/main/cpp/commands/IntakeOn.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/IntakeOn.h"
+
+IntakeOn::IntakeOn(double speed, Intake& intake)
```
Comment (duane701) [inline src/main/cpp/commands/IntakeOn.cpp]: Subsystem goes first. And you'll have to change the initializer list order.

```suggestion
IntakeOn::IntakeOn(Intake& intake, double speed)
```

File: src/main/cpp/RobotContainer.cpp [hunk 3]
```diff
@@ -21,6 +23,17 @@ RobotContainer::RobotContainer()
...
+  frc::SmartDashboard::PutData("Intake 10 percent", new IntakeOn(0.1, mIntake));
+  frc::SmartDashboard::PutData("Intake 20 percent", new IntakeOn(0.2, mIntake));
+  frc::SmartDashboard::PutData("Intake 30 percent", new IntakeOn(0.3, mIntake));
+  frc::SmartDashboard::PutData("Intake 40 percent", new IntakeOn(0.4, mIntake));
+  frc::SmartDashboard::PutData("Intake 50 percent", new IntakeOn(0.5, mIntake));
+  frc::SmartDashboard::PutData("Intake 60 percent", new IntakeOn(0.6, mIntake));
+  frc::SmartDashboard::PutData("Intake 70 percent", new IntakeOn(0.7, mIntake));
+  frc::SmartDashboard::PutData("Intake 80 percent", new IntakeOn(0.8, mIntake));
+  frc::SmartDashboard::PutData("Intake 90 percent", new IntakeOn(0.9, mIntake));
+  frc::SmartDashboard::PutData("Intake 100 percent", new IntakeOn(1.0, mIntake));
```
Comment (duane701) [inline src/main/cpp/RobotContainer.cpp]: Really? Could we buy a preference? They're pretty cheap.

No change required.

**Comment by duane701** [comment]
7e34805 should be squashed into the previous commit.

**Review by duane701 — CHANGES_REQUESTED** [review]

**Comment by duane701** [comment]
FTP

**Comment by toxicoreos** [comment]
PTAL

File: src/main/include/commands/IntakeOn.h [hunk 4]
```diff
@@ -7,10 +7,9 @@
 
 #pragma once
 
+#include <subsystems/Intake.h>
```
Comment (duane701) [inline src/main/include/commands/IntakeOn.h:10]: Our subsystem includes use `"`

```suggestion
#include "subsystems/Intake.h"
```

**Review by duane701 — CHANGES_REQUESTED** [review]
FTP

**Comment by duane701** [comment]
Replaced by #24

**Pull request closed without merging**